Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -2775,39 +2796,48 @@ CGContextRef CGBitmapContextCreate(void* data,
 
 /**
  @Status Caveat
- @Notes releaseCallback and releaseInfo is ignored.
- We only support formats that are 32 bits per pixel, colorspace and bitmapinfo that are ARGB.
+ @Notes If data is provided, it can only be in one of the few pixel formats Direct2D can render to in system memory: (P)RGBA, (P)BGRA, or Alpha8. If a buffer is
+        provided for a grayscale image, render operations will be carried out into an Alpha8 buffer instead.
+        Luminance values will be discarded in favour of alpha values.
 */
 CGContextRef CGBitmapContextCreateWithData(void* data,
                                            size_t width,
                                            size_t height,
                                            size_t bitsPerComponent,
                                            size_t bytesPerRow,
-                                           CGColorSpaceRef space,
+                                           CGColorSpaceRef colorSpace,
                                            uint32_t bitmapInfo,
                                            CGBitmapContextReleaseDataCallback releaseCallback,
                                            void* releaseInfo) {
     RETURN_NULL_IF(!width);
     RETURN_NULL_IF(!height);
-    RETURN_NULL_IF(!space);
+    RETURN_NULL_IF(!colorSpace);
 
-    // bitsperpixel = ((bytesPerRow/width) * 8bits/byte)
     size_t bitsPerPixel = ((bytesPerRow / width) << 3);
     WICPixelFormatGUID outputPixelFormat;
-    RETURN_NULL_IF_FAILED(_CGImageGetWICPixelFormatFromImageProperties(bitsPerComponent, bitsPerPixel, space, bitmapInfo, &outputPixelFormat));
+    RETURN_NULL_IF_FAILED(_CGImageGetWICPixelFormatFromImageProperties(bitsPerComponent, bitsPerPixel, colorSpace, bitmapInfo, &outputPixelFormat));
     WICPixelFormatGUID pixelFormat = outputPixelFormat;
 
+    void* actualBackingBuffer = data;
```

Comment:
There will be a future where we want to keep track of the user's input `data` and destroy it with their release callback, but we don't necessarily want the `CGIWICBitmap` to hold a reference to it.

I'd like to leave this in for that time, but maybe rename it to something more appropriate.